suite-native/video-assets/src/components/Video.tsx

```diff
@@ -52,16 +54,14 @@ export const Video = ({
                     style={applyStyle(activityIndicatorStyle)}
                 />
             )}
-            <ExpoVideo
-                style={applyStyle(videoStyle, { aspectRatio })}
-                source={videoSource}
-                shouldPlay
-                isMuted={isMuted}
-                isLooping
-                resizeMode={ResizeMode.CONTAIN}
-                onLoad={() => setIsLoading(false)}
-                {...restProps}
-            />
+            {!isLoading && (
```

Comment:
Is this fast enough to not cause any layout jumps? I am thinking about some empty View with same dimensions to prevent that.

Reply:
There is no layout shift happening. 

This is how it looks when loading: 
![Snímek obrazovky 2024-11-26 v 14 57 28](https://github.com/user-attachments/assets/e8c2e131-cf44-4556-83f7-2252186b64ee)

And when it loads:
![Snímek obrazovky 2024-11-26 v 14 57 43](https://github.com/user-attachments/assets/df1b5b55-fdc3-49de-b045-7b262282fd48)